stripe/src/main/java/com/stripe/android/model/ConfirmPaymentIntentParams.kt

```diff
@@ -511,13 +511,14 @@ data class ConfirmPaymentIntentParams internal constructor(
          */
         @JvmStatic
         internal fun createAlipay(
-            clientSecret: String,
-            returnUrl: String
+            clientSecret: String
         ): ConfirmPaymentIntentParams {
             return ConfirmPaymentIntentParams(
                 clientSecret = clientSecret,
                 paymentMethodCreateParams = PaymentMethodCreateParams.createAlipay(),
-                returnUrl = returnUrl
+                // return_url is no longer used by is still required by the backend
+                // TODO(smaskell): remove this when no longer required
+                returnUrl = "fake://return_url"
```

Comment:
nit, can we at least make it something like `stripe://return_url`?

Reply:
sure